client/lib/pdf-label-utils/index.js

```diff
@@ -45,5 +46,5 @@ export const getPrintURL = ( paperSize, labels, { labelsPrintURL, nonce } ) => {
 };
 
 export const getPreviewURL = ( paperSize, labels, { labelsPreviewURL, nonce } ) => {
-	return _getPDFURL( paperSize, labels, labelsPreviewURL, nonce );
+	return pdfSupport() ? _getPDFURL( paperSize, labels, labelsPreviewURL, nonce ) : null;
```

Comment:
Yes, I added this to not break everything in IE11 right now, but this line of code will soon get removed by #819. `pdfSupport()` is used in more places.